macros/schema_tests/column_values_basic/expect_column_values_to_have_consistent_casing.sql

```diff
@@ -0,0 +1,41 @@
+{% test expect_column_values_to_have_consistent_casing(model, column_name, display_inconsistent_columns=False) %}
+
+with test_data as (
+
+    select
+        distinct {{ column_name }} as distinct_values
+    from
+        {{ model }}
+
+ ),
+ {% if display_inconsistent_columns %}
+ validation_errors as (
+
+    select
+        lower(distinct_values) as inconsistent_columns,
```

Comment:
@agusfigueroa-htg FYI, I updated some of the formatting, so you'll have to pull those changes into your local branch